server/tests/api_v5/anonymous_account/test_account_create_send_validation_email.py

```diff
@@ -0,0 +1,111 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+from asyncio import sleep
+
+import pytest
+
+from parsec._parsec import anonymous_account_cmds
+from parsec.components.email import SendEmailBadOutcome
+from tests.common import Backend, HttpCommonErrorsTester
+from tests.common.client import AccountRpcClient
+from tests.common.letter_box import LetterBox
```

Comment:
missing import for `email_account_letterbox` fixture?

Reply:
lettter_box wasn't exposed in the __init__